### ISSUE TITLE
Room Setup

### ISSUE THREAD

**Issue by Harvey3141** [sergiobd/ViveTrackerUtilities#2]
Hi, 

This looks promising! 

Does this allow a room set up to be done with a headset plugged in? i.e can the calibration be done without ever need to plug a headset in?

**Comment by sergiobd** [comment]
> This looks promising!

Thanks, though I don't think I'll develop this further.

> Does this allow a room set up to be done with a headset plugged in? i.e can the calibration be done without ever need to plug a headset in?

No, but you can edit those settings manually in case you need to. I believe they are in Steam\config\chaperone_info.vrchap

**Comment by tiaoyuedexin** [comment]
Hi，Using your methods, I got the location of the tracker，but the coordinate scale of the tracker position is not appropriate.When the tracker moves a few centimeters, the position coordinates change by one point.I wonder if this is the result of no calibration?Does calibration have to be determined in room Settings?

**Comment by sergiobd** [comment]
@tiaoyuedexin, I don't think this is related to the room settings. Are you sure the scale parameters of your transform and its parent's are all set to 1?

**Comment by tiaoyuedexin** [comment]
Hello, thank you for your help, I don't know where to set these parameters?Do I need additional calibration?










At 2021-09-09 12:47:34, "Sergio Bromberg" ***@***.***> wrote:

@tiaoyuedexin, I don't think this is related to the room settings. Are you sure the scale parameters of your transform and its parent's are all set to 1?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
